pkg/manifests/bindata.go

```diff
@@ -163,6 +184,7 @@ func AssetNames() []string {
 var _bindata = map[string]func() (*asset, error){
 	"jsonnet/manifests/ceph-prometheus-rules.yaml": jsonnetManifestsCephPrometheusRulesYaml,
```

Comment:
Can we externalize this in some configuration file?

Reply:
auto-generated. We can't make persistent changes.